package.json

```diff
@@ -17,7 +17,7 @@
     "@typescript-eslint/parser": "^2.34.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-eslint-plugin": "^2.3.0",
-    "eslint-plugin-node": "^10.0.0",
+    "eslint-plugin-node": "^11.1.0",
```

Comment:
What do you think about using `eslint-plugin-n`?
The name of the plugin it depends on does not affect the name of the rules provided by eslint-plugin-mysticatea, so I think we can use another plugin.

https://github.com/eslint-community/eslint-plugin-mysticatea/blob/0f9086016c7c13cdaa7c6c9865834475c1642cb3/lib/foreign-rules/node.js#L7-L12

Reply:
My intention is to first update this plugin asap as much as possible with everything that's in here already & afterwards move to our own forks of the plugins once we've done so